packages/orchestrator/lib/abort.ts

```diff
@@ -0,0 +1,39 @@
+import type { Scheduler, Task } from '@nangohq/scheduler';
+import { validateTask } from './clients/validate.js';
+import { Ok, stringifyError } from '@nangohq/utils';
+import type { Result } from '@nangohq/utils';
+import { logger } from './utils.js';
+
+export async function scheduleAbortTask({ scheduler, task }: { scheduler: Scheduler; task: Task }): Promise<Result<Task | undefined>> {
+    const aborted = validateTask(task);
+    if (aborted.isOk()) {
```

Comment:
You are not rejecting of the validation doesn't pass?

Reply:
you are right. don't remember why i did that but that doesn't seem correct. will change to return an error if it doesn't validate